sormas-api/src/main/java/de/symeda/sormas/api/location/LocationReferenceDto.java

```diff
@@ -100,4 +100,35 @@ public static String buildCaption(
 		}
 		return stringBuilder.toString();
 	}
+
+	public static String buildCaption(String city, String street, String houseNumber, String additionalInformation) {
+
+		StringBuilder stringBuilder = new StringBuilder();
+
+		if (!DataHelper.isNullOrEmpty(city)) {
+			if (stringBuilder.length() > 0) {
+				stringBuilder.append(", ");
+			}
+			stringBuilder.append(city);
+		}
+		if (!DataHelper.isNullOrEmpty(street)) {
+			if (stringBuilder.length() > 0) {
+				stringBuilder.append(", ");
+			}
+			stringBuilder.append(street);
+		}
+		if (!DataHelper.isNullOrEmpty(houseNumber)) {
```

Comment:
I know it's the same as in the other buildCaption method, but there shouldn't be a comma in between street and house number if both are present; would be great if you could also fix that for the above method